requirements.txt

```diff
@@ -6,3 +6,6 @@ pandas
 seaborn
 sklearn
 tensorflow
+beautifulsoup4
```

Comment:
Pleas keep the requirements sorted alphabetically.  When this list gets long, that will make it easier to spot missing or duplicate entries.

Reply:
Changes have been made @cclauss , and thank you it is indeed a great way to avoid confusions later 👍.